docs/getting-started/dockstore-workflows.rst

```diff
@@ -68,41 +68,57 @@ GitHub Apps, Dockstore can react to changes on GitHub as they are made, keeping
 with GitHub automatically. To install our GitHub App on either a repository or organization,
 navigate to the ``/my-workflows`` page and click add workflow. Follow the steps for GitHub Apps and
 you will be redirected to GitHub where you can select which repositories to install the Dockstore
-GitHub app on.
+GitHub app on. The process is shown in the following images.
 
-In order for Dockstore to start pulling in content from GitHub, a ``.dockstore.yml`` file must be
-added to a branch of the repository that contains your workflow. This file contains information like
+.. figure:: /assets/images/docs/gh-app-reg-1.png
+   :alt: GitHub App Registration on Dockstore
+
+   Select Register Using GitHub Apps to get a link to our installation page
+
+.. figure:: /assets/images/docs/gh-app-reg-2.png
+   :alt: GitHub App Organizations Page
+
+   Select which organization to install our GitHub App on
+
+.. figure:: /assets/images/docs/gh-app-reg-3.png
+   :alt: GitHub App Organization Install Page
+
+   Install our GitHub App on either all repositories in an organization or a select few
+
+Once you've installed our GitHub app on a repository or organization, you'll need to add a ``/.dockstore.yml`` file to
+the root directory of a branch of the repository that contains your workflow. This file contains information like
```

Comment:
Is it always the root directory of the branch repository? Or the same directory as the primary descriptor? Some people put their descriptors in a sub directory for example.

Reply:
Good question, it must be in the root directory.

Reply:
Good to know, I'd suggest maybe changing the screen shot to having the primary descriptor in a sub directory and .dockstore.yml in root so its not ambiguous. Or maybe an additional screen shot explaining both cases.